packages/tabsheet/src/styles/vaadin-tabsheet-base-styles.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Copyright (c) 2022 - 2025 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import '@vaadin/component-base/src/style-props.js';
+import { css } from 'lit';
+import { loaderStyles } from '@vaadin/component-base/src/loader-styles.js';
+
+export const tabSheetStyles = [
+  loaderStyles,
+  css`
+    @layer base {
+      :host {
+        display: flex;
+        flex-direction: column;
+        border: var(--vaadin-tabsheet-border-width, 1px) solid
+          var(--vaadin-tabsheet-border-color, var(--vaadin-border-color));
+        border-radius: var(--vaadin-tabsheet-border-radius, var(--vaadin-radius-l));
```

Comment:
Should we also set something like this on the scroller to make focus-ring use border-radius?

```css
    border-bottom-left-radius: inherit;
    border-bottom-right-radius: inherit;
```

Reply:
Right, that would be nice attention to detail 👍 👍

Reply:
Done.